packages/components/pie-switch/src/index.ts

```diff
@@ -50,6 +51,8 @@ export class PieSwitch extends RtlMixin(LitElement) implements SwitchProps {
         this.handleFormAssociation();
     }
 
+    static shadowRootOptions = { ...LitElement.shadowRootOptions, delegatesFocus: true };
```

Comment:
This basically tells the component that if it receives a focus call, to pass that focus on to whatever the first focusable element in the template is. In this case, it is the input element. This means consumers can call focus on the input without having to know of the internal component template.

Reply:
reference: https://developer.mozilla.org/en-US/docs/Web/API/ShadowRoot/delegatesFocus